cmd/vic-machine/create/create.go

```diff
@@ -183,7 +185,7 @@ func (c *Create) Flags() []cli.Flag {
 		cli.StringSliceFlag{
 			Name:  "volume-store, vs",
 			Value: &c.volumeStores,
-			Usage: "Specify a list of location and label for volume store, e.g. \"datastore/path:label\" or \"datastore:label\".",
+			Usage: "Specify a list of location and label for volume store, nfs stores can have mount options specified as query parameters in the url target. \n\t Examples for a vsphere backed volume store are:  \"datastore/path:label\" or \"datastore:label\" or \"ds://my-datastore-name:store-label\"\n\t Examples for nfs back volume stores are: \"nfs://127.0.0.1/path/to/share/point?uid=1234&gid=5678&proto=tcp:my-volume-store-label\" or \"nfs://my-store/path/to/share/point:my-label\"",
```

Comment:
Minor, if you're going to use ODATA query parameters AND THEN use `:` to tokenize another parameter, why not just use the query parameter for that too?  I'm not a useability expert, but it's something to consider.  Check with @hickeng to get a sense of which direction to take here.  I'm not a fan of tokenizing URLs.